nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,71 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+tags:
+- SQL Server
+---
+
+The SQL Server transport allows endpoints to exchange messages using the Microsoft SQL Server Database Engine. It implements queues on top of database tables and send / receive operations as `INSERT` and `DELETE` operations.
+
+Please note that the SQL Server transport does not use or depend on SQL Server Service Broker.
```

Comment:
Ah, OK :smile:. I assumed you are just being very kind in your removal requests :wink:.